package.json

```diff
@@ -4,6 +4,7 @@
   "description": "Validate ProtoDef protocol definition in node",
   "main": "index.js",
   "scripts": {
+    "prepare": "git submodule update --init",
```

Comment:
Why do we need this? We don't have it in https://github.com/PrismarineJS/node-minecraft-data/blob/master/package.json

Reply:
By default git doesn't checkout the submodules without cloning repo recursively (github CI does this but not codespaces/gitpod), so you have to clone it manually after by running that. Adding to `prepare` just adds an extra command run after a local `npm install` which can init the submodules. But it's not needed though, just removed it.